pkg/dependencies/factory_test.go

```diff
@@ -20,6 +21,11 @@ type factoryTest struct {
 
 func newTest(t *testing.T) *factoryTest {
 	clusterConfigFile := "testdata/cluster_vsphere.yaml"
+	// Disable tools image executable for the tests
+	if err := os.Setenv("MR_TOOLS_DISABLE", "true"); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
just curious, why is this needed?

Reply:
If left enabled, it will try to use the cli tools image, pulling it during init. I don't want unit tests to make network calls